File: public/api/check_gd.py
"""
Check GD API - Python Version
Chuyển đổi từ check_gd.php
"""
from flask import jsonify

def check_gd():
    """Kiểm tra GD extension (trong Python không cần, dùng Pillow)"""
    try:
        from PIL import Image
        has_pil = True
    except:
        has_pil = False
    
    return jsonify({
        'gd_available': has_pil,
        'gd_info': {'PIL/Pillow': 'Available' if has_pil else 'Not installed'},
        'imagecreatefromstring': has_pil,
        'imagecreatetruecolor': has_pil
    })

File: public/api/check_gd.php
<?php
header('Content-Type: application/json');

$gd_info = [];
if (function_exists('gd_info')) {
    $gd_info = gd_info();
}

echo json_encode([
    'gd_available' => extension_loaded('gd'),
    'gd_info' => $gd_info,
    'imagecreatefromstring' => function_exists('imagecreatefromstring'),
    'imagecreatetruecolor' => function_exists('imagecreatetruecolor'),
]);
